Sources/SwifterSwift/UIKit/UIButtonExtensions.swift

```diff
@@ -167,34 +167,72 @@ public extension UIButton {
         states.forEach { setTitle(title, for: $0) }
     }
 
-    /// SwifterSwift: Center align title text and image
+    /// SwifterSwift: Set image on the left text's spacing
+    ///
     /// - Parameters:
-    ///   - imageAboveText: set true to make image above title text, default is false, image on left of text
     ///   - spacing: spacing between title text and image.
-    func centerTextAndImage(imageAboveText: Bool = false, spacing: CGFloat) {
-        if imageAboveText {
-            // https://stackoverflow.com/questions/2451223/#7199529
-            guard
-                let imageSize = imageView?.image?.size,
-                let text = titleLabel?.text,
-                let font = titleLabel?.font else { return }
-
-            let titleSize = text.size(withAttributes: [.font: font])
-
-            let titleOffset = -(imageSize.height + spacing)
-            titleEdgeInsets = UIEdgeInsets(top: 0.0, left: -imageSize.width, bottom: titleOffset, right: 0.0)
-
-            let imageOffset = -(titleSize.height + spacing)
-            imageEdgeInsets = UIEdgeInsets(top: imageOffset, left: 0.0, bottom: 0.0, right: -titleSize.width)
-
-            let edgeOffset = abs(titleSize.height - imageSize.height) / 2.0
-            contentEdgeInsets = UIEdgeInsets(top: edgeOffset, left: 0.0, bottom: edgeOffset, right: 0.0)
-        } else {
-            let insetAmount = spacing / 2
-            imageEdgeInsets = UIEdgeInsets(top: 0, left: -insetAmount, bottom: 0, right: insetAmount)
-            titleEdgeInsets = UIEdgeInsets(top: 0, left: insetAmount, bottom: 0, right: -insetAmount)
-            contentEdgeInsets = UIEdgeInsets(top: 0, left: insetAmount, bottom: 0, right: insetAmount)
-        }
+    func setImageLeftText(spacing: CGFloat) {
+        let insetAmount = spacing / 2
+        imageEdgeInsets = UIEdgeInsets(top: 0, left: -insetAmount, bottom: 0, right: insetAmount)
+        titleEdgeInsets = UIEdgeInsets(top: 0, left: insetAmount, bottom: 0, right: -insetAmount)
+    }
+
+    /// SwifterSwift: Set image on the right text
+    ///
+    /// - Parameters:
+    ///   - spacing: spacing between title text and image.
+    func setImageRightText(spacing: CGFloat) {
```

Comment:
Update CHANGELOG too.
```suggestion
    func setImageRightOfText(spacing: CGFloat) {
```